### PR TITLE
Pull request for code review - Portfolio mobile view structure

### PR DESCRIPTION
**Pull request by wickathou**
This is a minimal portfolio design. I tried to keep the design of the portfolio simple to highlight the work that will be featured on it.

This design takes the structure from the template 1 on the figma file with a few changes on the icons, background and font sizing.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="fill-100">
+    <nav class="navigation">
+      <span>Logo</span>
```
Comment (OyePriscilla) [inline index.html:13]: - [ ] Please take note that the logo suppose to have a link (a tag). This is for three reasons: convention, semantics, and user experience(UX). 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      <span>Menu</span>
```
Comment (OyePriscilla) [inline index.html:14]: - [ ] This suppose to be an icon or image, kindly work towards that please ✋.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    </nav>
+  </header>
+  <main class="fill-100">
+    <article>
+      <section>
+        <p>I'm Javi,</p>
+        <h1> A Web Designer that can code</h1>
```
Comment (OyePriscilla) [inline index.html:21]: - [ ] You can wrap these two elements with a single `h1` tag I guess. 😄

File: index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+        <p>I make ads and websites, combined with code to improve the conversion rates of their marketing</p>
+      </section>
+      <section class="pv">
+        <a href="http://">Let's connect</a>
+      </section>
+      <section>
+        <span>Link</span>
+        <span>Link</span>
+        <span>Link</span>
+      </section>
```
Comment (OyePriscilla) [inline index.html:33]: - [ ] Please kindly rearrange your lines of code, you only need just one section element to wrap all other elements. ℹ️

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @wickathou 😃, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

so-close-so-close-seth-meyers

## Highlights
Github flow observed✔️
Linters checks passed ✔️ 

## Required Changes ♻️

- [ ] Please I noticed that your design is not according to any of the specified [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=zdWk678TEA41AIuN-0). Kindly work with the any of the five designs. ℹ️ 

## Your Design
G1

## One of Figma Designs
crr16

- [ ] Kindly ensure that your PR summary is more descriptive.
- [x] Kudos for working on your Readme file, but you need to work more on it and ensure that all the necessary details are customized. This will help non-professional to quickly understand what your project is all about. 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+            <a href="http://">
+              <img class="icon" src="assets/images/github.svg" alt="Github profile @wickathou">
+            </a>
+          </span>
+          <span>
+            <a href="http://">
+              <img class="icon" src="assets/images/instagram.svg" alt="Linkedin profile @digijavi">
+            </a>
+          </span>
+        </div>
```
Comment (Owusu-Desmond) [inline index.html:49]: - [OPTIONAL] Please make sure that your social media links are displayed using list items. The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)
  - for example
  ```css
  <ul>
   <li>
     <a><img src="./images/Linkedin icon.png" alt="none"></a>
   </li>
  </ul>
  ```

**Review by nedjwab — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @wickathou   👋

Your project is complete! ✔️ 🏁

Your code was very professional 💯

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉

 
## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**